### DISCUSSION TITLE
I can create a pasword in coinbase apis

### DISCUSSION THREAD

**Discussion by antuun** [community/community#36195] [Apps, API and Webhooks] [answered]
Hello l need your help l can't create a pasword in coinbase apis

**Comment by jfullstackdev** [comment, marked as answer]
maybe this 

https://help.coinbase.com/en/exchange/managing-my-account/how-to-create-an-api-key

**Comment by antuun** [comment]
Thank you so much Have good day.
Everithig is ok thank you for your súper support.🤓🤡
